website/data/cloud-docs-nav-data.json

```diff
@@ -162,7 +162,8 @@
           { "title": "Overview", "path": "workspaces/settings" },
           { "title": "VCS Connections", "path": "workspaces/settings/vcs" },
           { "title": "Access", "path": "workspaces/settings/access" },
-          { "title": "Drift Detection", "path": "workspaces/settings/drift-detection" },
+          { "title": "Health Assessments", "path": "workspaces/settings/health-assessments" },
+          { "title": "Drift Detection", "path": "workspaces/settings/drift-detection", "hidden": true },
```

Comment:
I'd remove the page entirely and just do the redirect, but lmk if you feel very strongly that we shouldn't do that.